.github/workflows/ci.yml

```diff
@@ -15,14 +15,14 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python: ['3.7', '3.8', '3.9', '3.10']
-        django: ['3.2', '4.0']
-        mozilla_django_oidc: ['1.2', '2.0']
+        python: ['3.8', '3.9', '3.10', '3.11']
+        django: ['3.2', '4.1', '4.2']
+        mozilla_django_oidc: ['2.0', '3.0']
```

Comment:
this requires a documentation update explaining that you need at least `mozilla-django-oidc 3.0` for Django 4.1+

I checked and it seems we cannot express this in `setup.cfg`